app/concepts/matestack/ui/core/del/del.haml

```diff
@@ -0,0 +1,5 @@
+%del{@tag_attributes.merge({:cite => options[:cite], :datetime => options[:datetime]})}
```

Comment:
I'd prefer the `@tag_attributes.merge([...])` stuff to be within a setup method in the del.rb, but that's a personal preference. Should be fine like this for now.